### ISSUE TITLE
GW model variables not declared when outputting

### ISSUE THREAD

**Issue by penguian** [CABLE-LSM/CABLE-Trac#210]
`keyword_nogit` `owner:bep599@nci.org.au` `resolution_fixed` `type_defect`   |    by amu561

___

Several GW parameters (slope, slope_std and GWdz) should be in a gw_model IF-block in cable_output.F90. They currently cause a seg fault when turning groundwater off.

Line 1231 onwards in CMIP6-MOSRS:
```
IF(output%params .OR. output%slope) CALL define_ovar(ncid_out, opid%slope,   &
           'slope', '-', 'Mean subgrid topographic slope', &
	    patchout%slope, 'real', xID, yID, zID, landID, patchID)
```

___
_Issue migrated from [trac:210](https://trac.nci.org.au/trac/cable/210) at 2023-11-27 11:27:47 +1100_

**Comment by penguian** [comment]
_@jxs599@nci.org.au_ _changed status from `new` to `closed`_

**Comment by penguian** [comment]
_@jxs599@nci.org.au_ _changed resolution from `` to `fixed`_

**Comment by penguian** [comment]
_@jxs599@nci.org.au_ _changed milestone from `` to `1. Closed`_

**Comment by penguian** [comment]
_@jxs599@nci.org.au_ _commented_


___
resolved under Ticket #208

**Comment by penguian** [comment]
_@ccc561@nci.org.au_ _set keywords to `nogit`_
